cli/cli.go

```diff
@@ -199,3 +203,16 @@ func (c *CLI) WaitForActions(ctx context.Context, actions []*hcloud.Action) erro
 
 	return nil
 }
+
+func (c *CLI) GetServerName(id int) string {
+	if len(c.serverNames) == 0 {
```

Comment:
```suggestion
	if c.serverNames == nil {
```

Eliminates false-positives when `len(c.serverNames)` is indeed 0 because there are no servers in the project.